newdocs/typeclass-cheatsheet.md

```diff
@@ -0,0 +1,223 @@
+# Consensus Cheatsheet (Type Families, Classes, & Notes)
+
+See [Diagrammatic Conventions](#diagrammatic-conventions)
+    
+## Birds Eye Overview of Consensus Types
```

Comment:
It's a pleasant surprise that the `haskell` syntax lighting works so well even within the diagram 😄

Reply:
yep!